### ISSUE TITLE
Suggestion Glowing NPC'S

### ISSUE THREAD

**Issue by Sprungente** [FlyingPikachu/Quests#294]
hello Quester! Hello Pikachu ! 

Today i have (i think so) a great idea. 
To have player easier to understand quests, some NPC's can i special situation glowing. 
So every NPC, the player has to talk with him, is glowing until he talked with him. 
It also useful for delivery stages.  

What do you thinking about my idea? Do you have additional ideas?

**Comment by FlyingPikachu** [comment]
This is already possible via Citizens (see the _/npc glowing_ command).

**Comment by Sprungente** [comment]
sure sure, you don't understand what i mean. I don't want an npc who glowing all time. Only when it is a goal for a interaction like delivery or talking. It should be clearer for users. 

and make a favor: don't close it after reading. get little room for discuss

**Comment by FlyingPikachu** [comment]
I see. Although it may (or may not) be possible to toggle whether an NPC is glowing via the Citizens API, I personally believe it's beyond the scope of Quests to play with that sort of integration. Even if we did, we'd have to account for what happens if too many NPCs are glowing at once (whioch would potentially induce lag, like your particle suggestion in #295), what happens if the server suddenly crashes (assuming NPCs retain their glow state through restarts), and how different users may want to use the glow (we would essentially have to provide every functionality that the Citizens command does, such as size and color... although I suppose we could just have the user enter the Citizens command to pass to Citizens itself, but at that point one might as well just write a script for this).

In summary, what you are asking is a lot of work for little gain, with no guarantee that it even work at all. If you can provide a link to the Citizens method for controlling glow in their Javadoc or are willing to code a bit yourself, I'll consider re-opening this.
